test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -171,20 +171,11 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
             $observed | should be $currentVersion
         }
 
-        It "-Version should ignore other parameters" {
-            $currentVersion = "powershell " + $PSVersionTable.GitCommitId.ToString()
-            $observed = & $powershell -version -command get-date
-            # no extraneous output
-            $observed | should be $currentVersion
-        }
-
         It "-Version should write an error if a value is present" {
             $versionValue = "abrakadabra"
-            $tempFile = Join-Path $testdrive "expectedError.txt"
-            $observed = & $powershell -version $versionValue > $tempFile
-            $expectedError = (Get-Content $tempFile)[0]
-
-            $expectedError | Should Match $versionValue
+            $observed = & $powershell -version $versionValue 2>&1
```

Comment:
😕 I skipped [Feature] in previous PR. I saw "CI" on top the test file but we have "Feature" tests below. 
 _We need an idea how exclude this in future._
(I wonder that the test was passed locally.)

In CI log I see messages:
```
[00:08:17] Usage of '-Version -command' is not supported. '-Version' currently only returns the current PowerShell version.
[00:08:17] +
[00:08:17] Usage of '-Version abrakadabra' is not supported. '-Version' currently only returns the current PowerShell version.
```
The first message looks incorrect. Maybe: "Error in '-Version {0}'. Now '-Version' shows the current PowerShell version and don't accept an argument."

Also previous test helped us to catch the problem - so I prefer to don't remove it because it may test another code path in future.

Reply:
The test I removed was when we supported:

```bash
poweshell -version -command 1+1
```

I think this syntax doesn't make sense as what it ended up doing was outputting the version string and ignoring everything after it.  It seems that this should have caught an unintentional user error which is why I removed that test since the new behavior prevents that old behavior.

I agree that the error message could be better particularly in the above scenario.  I'll update it.

Reply:
I think I'll change it to

```none
Unsupported value '{0}' passed to `-Version`. '-Version' currently only returns the current PowerShell version and must be used standalone.
```